docs/projects/pulsar_lvds/index.rst

```diff
@@ -0,0 +1,346 @@
+The SAR architecture allows unmatched performance both in noise and in linearity.
+The :adi:`AD7960` / :adi:`AD7961` contains a low power, high speed, 18-bit/16-bit
+sampling ADC, an internal conversion clock, and an internal reference buffer.
+
+On the CNV± edge, the AD7960 samples the voltage difference between the IN+ and
+IN− pins. The voltages on these pins swing in opposite phase between 0 V and 
+4.096 V and between 0 V and 5 V. The reference voltage is applied to the part
+externally. All conversion results are available on a single LVDS self clocked
+or echoed clock serial interface. The AD7960 is available in a 32-lead LFCSP
+(QFN) with operation specified from −40°C to +85°C
+
+Applications:
+  * Digital imaging systems
+  * High speed data acquisition
+  * High dynamic range telecommunications receivers
+  * Spectrum analysis
+  * Test equipment
+
+Supported boards
+-------------------------------------------------------------------------------
+
+-  :adi:`EVAL-AD7625-FMCZ <AD7625>`
+-  :adi:`EVAL-AD7626-FMCZ <AD7626>`
+-  :adi:`EVAL-AD7960-FMCZ <AD7960>`
+-  :adi:`EVAL-AD7961-FMCZ <AD7961>`
+
+Supported devices
+-------------------------------------------------------------------------------
+
+-  :adi:`AD7625`
+-  :adi:`AD7626`
+-  :adi:`AD7960`
+-  :adi:`AD7961`
+-  :adi:`ADR3412`
+-  :adi:`ADR4520`
+-  :adi:`AD74540`
+-  :adi:`AD4550`
+-  :adi:`AD8031`
+-  :adi:`ADA4899-1`
+-  :adi:`ADA4897-1`
+-  :adi:`ADP7102`
+-  :adi:`ADP7104`
+-  :adi:`ADP124`
+-  :adi:`ADP2300`
+
+Supported carriers
+-------------------------------------------------------------------------------
+
+-  :xilinx:`ZedBoard <products/boards-and-kits/1-8dyf-11.html>` on FMC slot
+
+Block design
+-------------------------------------------------------------------------------
+
+Block diagram
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The data path and clock domains are depicted in the below diagram:
+
+.. image:: pulsar_lvds_block_diagram.svg
+   :width: 800
+   :align: center
+   :alt: PulSAR LVDS block diagram
+
+Configuration modes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The RESOLUTION_16_18N configuration parameter defines the resolution of the ADC
+(16 or 18  bits). By default it is set to 0. Depending of the project, some
+hardware modifications need to be done on the board and/or make command:
+
+In case of the **AD7960/AD7961** project:
+
+.. code-block::
+
+   make RESOLUTION_16_18N=0
+
+In case of the **AD7625/AD7626** project:
+
+.. code-block::
+
+   make RESOLUTION_16_18N=1
+
+Jumper setup AD7625/AD7626
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+================== ================= ===========================================
+Jumper/Solder link Default Position  Description
+================== ================= ===========================================
+LK2                Inserted          Connects REFIN to the 1.2V external
```

Comment:
I think paragraphs should end with a period `.`, or remove it from row `LK6` at line 121

Reply:
Done in V2.